### PR TITLE
Bookstore: Feature setup

### PR DESCRIPTION
**Pull request by gitbygaurav**
### Hey CR 😊

AidanHelloAidanGallagherHelloGIF

In this PR, I have :

- Initialised React app.
- Created a directory for components.
- Added [React Router](https://reactrouter.com/web/guides/quick-start) and set two Routes and Links for the app's navigation:
   - Books - the default view:
     - Display's the list of books.
       - Created a component called Book for displaying a single book (receives a title and an author as a prop) and reused it in a component that displays a list of books. 
     -  Display's a form for adding a book (no functionality yet). 
       - Created a separate component for this form.
   - Categories:
     - Displayed a button "Check status" only.
- Given some basic styling with tailwind css.  
         

Thanks 🍻
Gaurav

### CODE REVIEW

File: src/Pages/BooksPage.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import AddBookForm from '../components/AddBookForm';
+
+const BooksPage = () => {
+    return (
+        <div className='border-2 border-black mt-5'>
+            <BooksList />
+            <AddBookForm />
+        </div>
+    )
+}
```
Comment (danmainah) [inline src/Pages/BooksPage.jsx]: - [ ] @gauravgangwarweb great work on your component however am getting the error below, its caused by having a return statement yet you don't have any methods inside the component,
``` suggestion
const BooksPage = () =>  (
        <div className='border-2 border-black mt-5'>
            <BooksList />
            <AddBookForm />
        </div>
    )
```
image

File: src/Pages/CategoriesPage.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CategoriesPage = () => {
+    return (
+        <div>
+            <button className='border-2 border-black' type='button'>Check status</button>
+        </div>
+    )
+}
```
Comment (danmainah) [inline src/Pages/CategoriesPage.jsx]: - [ ]  @gauravgangwarweb great work on your component however am getting the error below, its caused by having a return statement yet you don't have any methods inside the component, kindly check my suggestion in the BooksPage component

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @gauravgangwarweb , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by gitbygaurav** [comment]
Hii @danmainah 👋

ThankYouStickerThanksStickerGIF

Thanks for reviewing my PR. I will make the required changes as suggested by you 😊.

**Review by jasonfelice — APPROVED** [review]
Hi  @gauravgangwarweb 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gitbygaurav** [comment]
Hii @jasonfelice 😊

Thanks for reviewing and approving my PR.

ThankYouStickerThanksStickerGIF (2)

**Pull request merged**